subsys/bluetooth/mesh/shell.c

```diff
@@ -23,11 +24,17 @@
 #include "transport.h"
 #include "foundation.h"
 #include "settings.h"
+#include "access.h"
 
 #define CID_NVAL   0xffff
 
 static const struct shell *ctx_shell;
 
+#define shell_print_ctx(_ft, ...) \
+		if (ctx_shell != NULL) { \
+			shell_print(ctx_shell, _ft, ##__VA_ARGS__); \
+		}
```

Comment:
Needs to be surrounded by `do { /* ... */ } while (0), according to coding guidelines: https://www.kernel.org/doc/html/v4.10/process/coding-style.html#macros-enums-and-rtl